src/linting/checks.jl

```diff
@@ -416,8 +416,9 @@ function collect_hints(x::EXPR, missing = true, isquoted = false, errs = Tuple{I
         # collect parse errors
         push!(errs, (pos, x))
     elseif !isquoted
-        if missing && CSTParser.isidentifier(x) && !hasref(x) && !(valof(x) == "var" && parentof(x) isa EXPR && typof(parentof(x)) === CSTParser.NONSTDIDENTIFIER)
-            
+        if missing && CSTParser.isidentifier(x) && !hasref(x) && 
+            !(valof(x) == "var" && parentof(x) isa EXPR && typof(parentof(x)) === CSTParser.NONSTDIDENTIFIER) &&
+            !(valof(x) == "stdcall" && is_in_fexpr(x, x->typof(x) === CSTParser.Call && isidentifier(x.args[1]) && valof(x.args[1]) == "ccall"))
```

Comment:
Do we handle the other calling conventions as well? The original issue had a link to the Julia parser file that had the names of all the calling conventions that can come up.

Reply:
Sorry, I hadn't clicked through - fixed